app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,4 @@
+class DashboardController < ApplicationController
```

Comment:
I created this dashboard to get the tests to pass, but we can always change later if it's not what we need.

Reply:
Fine by me